atdll/atdll.cpp

```diff
@@ -669,8 +771,16 @@ AT_HandleRawInput(WPARAM *wParam, LPARAM *lParam)
         return false;
     }
 
+    g_lastDevice = hdr.hDevice;
     debugf("Got HID raw input event for device %p", hdr.hDevice);
 
+    // Load the calibration info on startup
+    if (!g_hasCalibrationLoaded) {
```

Comment:
Oh god, actually this is a lot uglier than I thought (I forgot you don't have a reference to any touchpads on startup). I take back what I said, it would probably be better to wait until we have a way to persistently identify touchpads to do this (so we can actually do this on startup instead of here). Sorry for the back and forth.

Reply:
Haha no worries. I've removed that bit and pushed the changes.